scene/main/node.cpp

```diff
@@ -3671,7 +3708,12 @@ void Node::_bind_methods() {
 		ClassDB::bind_vararg_method(METHOD_FLAGS_DEFAULT, "rpc_id", &Node::_rpc_id_bind, mi);
 	}
 
+#ifndef DISABLE_DEPRECATED
 	ClassDB::bind_method(D_METHOD("update_configuration_warnings"), &Node::update_configuration_warnings);
+#endif
+#ifdef TOOLS_ENABLED
+	ClassDB::bind_method(D_METHOD("update_configuration_info"), &Node::update_configuration_info);
```

Comment:
Binding the method like this will make the script break in exported project. Any exposed method in non-editor class should exist in non-tool builds (usually as empty method).

Reply:
Updated this one + the virtual `_get_configuration_info` method bind.